commercial/app/views/hosted/hostedArticleOnward.scala.html

```diff
@@ -1,5 +1,5 @@
 @import common.commercial.hosted.HostedPage
-@(trails: Seq[HostedPage])(implicit request: RequestHeader)
+@(trails: Seq[HostedPage], defaultRows: Int = 2, maxRows: Int = 4)(implicit request: RequestHeader)
```

Comment:
I would take out the default values as there aren't that many places where it's used.  Would be clearer to have to specify them in the calling code.